### PR TITLE
[styles] Improve visibility and findability of highway services and rest areas

### PR DESCRIPTION
**Pull request by pastk**
Sometimes a fuel station is not mapped in a service area, in this case its helpful to see the service area icon on earlier zooms (I made it the same as fuel stations).
If a fuel station is mapped then its icon will have a priority over the service area icon (there are cases when a fuel station is tagged in detail, but a corresponding service area doesn't have a name even).

Its strange, but highway-services/rest_area were missing in the vehicle style completely - added.

Also fixed broken highway-rest_area captions display.

Btw, it'd be nice to have a better icon for highway-services, because they're mostly about fuel and food/drinks, but we're using a "car repair" icon atm. Anyone who is keen to find or draw a better icon?

### CODE REVIEW

**Review by vng — APPROVED** [review]
You didn't duplicate the same for node[highway=*]
Is it ok? I suppose there is no difference now, but maybe in future..

**Comment by pastk** [comment]
> You didn't duplicate the same for node[highway=*] Is it ok? I suppose there is no difference now, but maybe in future..

Yeap there is no difference

**Comment by pastk** [comment]
And in the future I hope we won't have this node|area duplication in styles

File: data/categories.txt [hunk 1]
```diff
@@ -11692,11 +11692,11 @@ sk:Garáž
...
 
-highway-rest_area
-en:Highway Rest Area
+highway-rest_area|highway-services
+en:4Rest Area
 en-GB:Lay By|rest area
 en-AU:Rest Area|lay by
-ru:Зона отдыха на трассе
+ru:Зона отдыха|Место отдыха
 bg:Почивка|магистрала|Почивна зона
```
Comment (biodranik) [inline data/categories.txt:11700]: be = Зона адпачынку

File: data/categories.txt [hunk 2]
```diff
@@ -11729,6 +11729,24 @@ el:Χώρος ανάπαυσης αυτοκινητιστών
 sk:Odpočívadlo
 mr:महामार्ग विश्रांती क्षेत्र
 
+highway-services
+en:4Service Area|service station|travel plaza
```
Comment (biodranik) [inline data/categories.txt:11733]: be = Зона абслугоўвання

**Review by biodranik — APPROVED** [review]
That is very important for car drivers.

Can you please rebase?

**Pull request merged**